Cargo.toml

```diff
@@ -32,6 +31,13 @@ pessimistic-proof = { path = "crates/pessimistic-proof" }
 pessimistic-proof-core = { path = "crates/pessimistic-proof-core" }
 pessimistic-proof-test-suite = { path = "crates/pessimistic-proof-test-suite" }
 
+agglayer-prover = { git = "https://github.com/agglayer/provers.git", features = [
+    "testutils",
+], branch = "feature/rework-agglayer-prover-types" }
```

Comment:
Just noting the target branch should change once the feature lands.

Also, can we make the testutils feature only enabled for testing code? This makes it enabled globally. I see some other crates enable it explicitly in dev-deps.